packages/syft/tests/syft/security/steal_objects_test.py

```diff
@@ -27,6 +27,7 @@ def test_steal_data_through_mutation(
     assert bag[0] == 1
 
     # and then mutate it with someone elses secret data
+    # stdlib
```

Comment:
TODO: remove this.

Reply:
@shubham3121 , are we clear with those questions we earlier had based on the algorithm?